.github/workflows/feature_deploy.yml

```diff
@@ -37,559 +37,151 @@ jobs:
     - name: Initialise Environment
       run: echo "Setting up ${{ env.environment }} environment"
 
-  #################### 1. PORTAL UI ####################
-  deploy-portal-ui:
+  deploy-webapps:
     needs: deployment-environment
-    name: Portal UI
+    name: Deploy Webapps
     runs-on: [self-hosted, linux, deployment]
     # TODO: FN-1077 NOTE: we don't currently have a `feature` GitHub environment yet so use the `dev` one while we're not doing much.
     # Restore the parameterised version throughout!
     environment: dev
     # environment:
     #   name: ${{ needs.deployment-environment.outputs.environment }}
-    env:
-      repository: portal-ui
-      
+    strategy:
+      max-parallel: 4
+      matrix:
+        repository: 
+          - portal-ui
+          - portal-api
+          - gef-ui
+          - trade-finance-manager-ui
+          - trade-finance-manager-api
+          - dtfs-central-api
+          - external-api
+        include:
+          - path: portal
+            repository: portal-ui
```

Comment:
We need this as the `portal-ui` webapp lives in the `portal` folder. However, GHA has quite a neat augmentation mechanism.